Case studies into develop (#399)
b2io/base2.io: _content/caseStudies/01-color-snap.md

---
client: 'Sherwin-Williams'

> excerpt: 'Working with our design partner Resource/Ammirati, we developed an application for Sherwin-Williams that allows paint professionals and every day consumers to shop for paint and experience color like never before.'

Reviewer: @mewdriller should we continue with the name "Resource/Ammirati"? That was the name at the time, but now they're "IBM iX"

Reviewer 2: Maybe a parenthetical mention that they're now part of IBM iX?

highlights:

>   - Implements a complex selection tool that allows users to edit and customize their uploaded photos with any Sherwin-Williams paint color.

Reviewer: The term "implements" is coming from the developer/application perspective where the other bullets are more about the user actions available.

```suggestion
  - Use a complex selection tool to edit and customize uploaded photos with any Sherwin-Williams paint color.
```

  - Choose from pre-existing color palettes or create a custom palette.
  - Save color choices or scenes to revisit or purchase.
  - Explore the color wall to find new colors and combinations.
link: 'https://www.sherwin-williams.com/visualizer'

> linkText: 'sherwin-williams.com/visualizer'

Reviewer: ❓ I wonder if we should have separate link text or always use the project as the link text (rather than having a separate line for the link under the product name.

logo: 'img/clients/sherwin-williams-logo-white.png'
path: 'color-snap'
project: 'Color Snap Visualizer'

> technologies: ['react', 'redux', 'sass', 'jasmine', 'karma']

Reviewer: 💬 We'll need to update the tech list. @mewdriller will know better than me, but I believe the list could be angularJS, sass, jasmine, karma

Reviewer 2: That's correct, the list should be angular, sass, jasmine, and karma.
